llvm/lib/Target/AMDGPU/SIInstructions.td

```diff
@@ -3338,14 +3338,17 @@ def : GCNPat <
                         (as_i1timm $bound_ctrl))
 >;
 
-def : GCNPat <
-  (i64 (int_amdgcn_update_dpp i64:$old, i64:$src, timm:$dpp_ctrl, timm:$row_mask,
+class UpdateDPP64Pat<ValueType vt> : GCNPat <
```

Comment:
Nit: instead of creating a class you could do this with `foreach vt = [i64, f64] in ...`

Reply:
Done